src/components/caption/_index.scss

```diff
@@ -0,0 +1,17 @@
+@use "../../scss";
+
+.c-caption {
+	@include scss.component-properties("caption");
+
+	.c-caption__title {
+		@include scss.component-properties("caption-title");
```

Comment:
is this naming scheme ok `caption-title` for component properties?

Reply:
Yeah this works and it's something that has been done in other components